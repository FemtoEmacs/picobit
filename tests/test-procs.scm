;; test processes

(start-first-process 
 (lambda ()
   (let loop ((a 1)) 
     (begin
       (spawn 
        (lambda ()
          (let loop2 ((a2 0)) 
            (if (= a a2) (exit)
                (loop (+ a2 1))))))
       (if (= a 10) 
           '()
           (loop (+ a 1)))))))
 