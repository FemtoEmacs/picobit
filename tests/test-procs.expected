:0080000080
:10800000FBD700020051005002B4001CB582EED82C
:1080100001D5510000CD504040D12070000002B485
:108020000023710120B000DA02B4002D710120217B
:1080300002CDB40047B58AEE0521B581710120CE8D
:10804000EF022022D000EF01212102CDCDB4008724
:10805000B58EEE200EC79202EF2005C122B55E71EB
:1080600001D52102CDB4006971012040CFCD22029B
:10807000CDB400757101212102CDCDB4007F700017
:108080004022D14121D6EF0000212302CDCDB40002
:1080900092710120B5C4222402CDCDB4009F71019C
:1080A00020222402CDCDB400B1B516EE0421B50DC9
:1080B00071012221C7B89BB6800040CF02B400C135
:1080C00071012040D393D800EF4021CFD120CE00C2
:0F80D000D7EF2005C122B0003D71012002CDEF96
:00000001FF
